app/src/main/java/com/sun/music_64/utils/Constants.java

```diff
@@ -11,12 +11,28 @@
             = "https://api.soundcloud.com/tracks/%d/download?client_id=%s";
     public static final String BASE_STREAM_URL
             = "https://api.soundcloud.com/tracks/%d/stream?client_id=%s";
-    public static final String KIND_TREND = "trend";
+    public static final String KIND_TREND = "trending";
     public static final String GENRES_ALL_MUSIC = "soundcloud:genres:all-music";
```

Comment:
15 đến 20 những String này sẽ dùng ở đâu em?

Reply:
Em dùng lúc click vào thể loại thì em truyền "loại genres" sang màn hình list để load API. Phần này   
là create UI nên em chỉ dùng mặc định "allmusic"

Reply:
em có thấy đang lặp nhiều lần soundcloud:genres không
với file Constant đang phìng to. Em có thể dùng StringDef theo cách của bạn Cảnh nhé. 
Tạo một file GenreEntity chẳng hạn